opentelemetry-etw-logs/src/exporter/mod.rs

```diff
@@ -90,33 +96,36 @@ impl ETWExporter {
 
         let event_tags: u32 = 0; // TBD name and event_tag values
         let field_tag: u32 = 0;
-        let mut event = tld::EventBuilder::new();
-
-        // reset
-        event.reset(
-            self.options.get_etw_event_name(log_record),
-            level,
-            Self::KEYWORD,
-            event_tags,
-        );
 
-        event.add_u16("__csver__", 1024, tld::OutType::Unsigned, field_tag); // 0x400 hex
+        EVENT_BUILDER.with(|event_builder| {
```

Comment:
nit: Using `with_borrow_mut` would be more concise. You wouldn't need the `event_builder.borrow_mut();` call then.

```suggestion
        EVENT_BUILDER.with_borrow_mut(|event_builder| {
```